src/benchmarks/pmemobj_gen.cpp

```diff
@@ -542,8 +542,13 @@ pobj_direct_op(struct benchmark *bench, struct operation_info *info)
 	auto *bench_priv = (struct pobj_bench *)pmembench_get_priv(bench);
 	auto *pw = (struct pobj_worker *)info->worker->priv;
 	size_t idx = bench_priv->obj(info->index);
-	if (pmemobj_direct(pw->oids[idx]) == nullptr)
-		return -1;
+	PMEMoid bad = {1, 1};
+	for (int i=0; i<1024; i++) {
```

Comment:
run this under `cstyle` ;)
Please `#define` all numbers in your code. It's not strictly necessary, but in this case will give you a place to add a comment on why this is needed and how to tune it.